ui/src/app/utils/countries.ts

```diff
@@ -0,0 +1,253 @@
+enum Country {
+  US = 'United States',
```

Comment:
US is the top of the list, because we want United States to show up as the first option.